lyrics/20-azlyrics.py

```diff
@@ -54,7 +54,7 @@ def html_to_text(s):
 
 artist = normalize_parameter(sys.argv[1])
 title = normalize_parameter(sys.argv[2])
-artist = re.sub("[^A-Za-z0-9]+", "", artist)
+artist = re.sub("[^A-Za-z0-9]+", "", artist[4:] if artist[:4] == "the " else artist)
```

Comment:
Having two unrelated operations in one line is hard to read. And to make it even simpler, use `str.removeprefix()` instead of reimplementing that method manually.